src/core/Cutter.cpp

```diff
@@ -656,7 +656,13 @@ bool CutterCore::loadFile(QString path, ut64 baddr, ut64 mapaddr, int perms, int
     }
 
     if (perms & RZ_PERM_W) {
-        rz_core_cmd0(core, "omfg+w");
+        RzPVector *maps = rz_io_maps(core->io);
+        void **it;
+        rz_pvector_foreach(maps, it)
+        {
+            auto *map = static_cast<RzIOMap *>(*it);
```

Comment:
```suggestion
            auto *map = reinterpret_cast<RzIOMap *>(*it);
```

or even more elegantly:
```c++
for (auto map : CutterPVector<RzIOMap>(maps)) {
    // ...
```